resources/assets/js/components/pages/admin/categories/editor/Descriptors.vue

```diff
@@ -196,9 +232,14 @@
                 descriptor.allowed_values = descriptor.allowed_values.split("|").join(", ");
                 return descriptor;
             });
+
             if (this.descriptors.length === 0) {
                 this.insertTempDescriptor();
             }
+
+            window.dsc = this;
+            window.onLoadedQueue = window.onLoadedQueue ? window.onLoadedQueue : [];
+            window.onLoadedQueue.push(this.setup);
```

Comment:
The load queue is a custom mechanism I implemented into this application a while ago that is emptied when global resources are done loading (APIs, mainly). This component needed this added now because as soon as it can it needs to make an API request.